corehq/motech/repeaters/models.py

```diff
@@ -268,6 +268,7 @@ class Repeater(RepeaterSuperProxy):
     is_paused = models.BooleanField(default=False)
     next_attempt_at = models.DateTimeField(null=True, blank=True)
     last_attempt_at = models.DateTimeField(null=True, blank=True)
+    max_workers = models.IntegerField(default=0)
```

Comment:
What do you think of spinning this feature off in a separate PR (if we decided it's needed)? We have a lot of knobs to turn already, and I wonder if we'll need this one?

Reply:
We needed this one years ago. OpenMRS integrations involve Events and Observations, and it is _really useful_ if we can send repeat records in the order in which their forms were submitted. More generally, setting `max_workers` to 1 allows us to ensure that repeat records are sent chronologically.

And setting it to [all of the workers] is an insurance policy for this PR to ensure that we can still handle the highest volume for repeaters that need it. I'd sleep easier if this is included.